README.md

```diff
@@ -1,2 +1,135 @@
-# aaq-core
-Ask-a-question service core service
+<pre align="center" style="text-align:center; font-size: 1vw; background:none;">
+    __            __        ______
+    /""\          /""\      /    " \
+    /    \        /    \    // ____  \
+    /' /\  \      /' /\  \  /  /    )  )
+  //  __'  \    //  __'  \(: (____/ //
+ /   /  \\  \  /   /  \\  \\         \
+ (___/    \___)(___/    \___)\"____/\__\
+ASK-A-QUESTION
+</pre>
+
+<p align="center" style="text-align:center">
+<a href="https://idinsight.github.io/aaq-core/">Docs</a> |
```

Comment:
When we say `Docs` do we mean `API Reference`? Should we name this better then? Like `Developer Documentation` or `Quick Start`